src/node-lib/curriculum-api-2023/queries/subjectListing/processLessons.test.ts

```diff
@@ -0,0 +1,106 @@
+import { processLessons } from "./processLesson";
+import { SubjectDataArrayRaw } from "./subjectListing.schema";
+
+describe("processLessons", () => {
```

Comment:
for future reference, you can get these fixtures from the oak curriculum library